Add handrolled schema helpers (#687)
ghostdogpr/caliban: vuepress/docs/docs/schema.md

The package of the generated code is derived from the folder of `outputPath`. This can be overridden by providing an alternative package with the `--packageName` option.

By default, each Query and Mutation will be wrapped into a `zio.UIO` effect. This can be overridden by providing an alternative effect with the `--effect` option.

## Building Schemas by hand

> Sometimes the for what ever reason schema generation fails. This can sometimes happen if your schema has co-recursive types and Magnolia is unable

Reviewer: ```suggestion
Sometimes for whatever reason schema generation fails. This can happen if your schema has co-recursive types and Magnolia is unable
```
